evalml/model_understanding/prediction_explanations/_user_interface.py

```diff
@@ -202,7 +202,7 @@ def make_dict(self, shap_values, normalized_values, pipeline_features, top_k, in
         return {"explanations": json_output}
 
 
-def _make_single_prediction_shap_table(pipeline, input_features, top_k=3, training_data=None,
+def _make_single_prediction_shap_table(pipeline, input_features, y, index_to_explain, top_k=3,
```

Comment:
Can you please add `index_to_explain` to the docstring?

Reply:
yup been working that!